core/src/main/java/com/graphhopper/util/Constants.java

```diff
@@ -66,6 +66,7 @@
     private static final int JVM_MAJOR_VERSION;
     private static final int JVM_MINOR_VERSION;
 
+    // todonow: make sure this version is compatible also for older imports!
```

Comment:
Do you think this is required? (similar question applies to the method where you kept the CH file names compatible)

We never kept backward compatibility for our files as they are recreated due to every OSM data update. I do not see value until we officially support offline routing support (maybe something for 3.0 ;) )

Reply:
Its probably not required because (?) I kept the old file names. I thought it would be ugly to change the filenames now and then change them again soon in #1708. Do you want to remove this complexity and just change the filenames now also? Otherwise we probably do not need to change the data versions, but I left this comment to double check.

Reply:
With my most recent commits this PR should be compatible with current master. On the other hand there is string manipulation logic to achieve this. I think its ok like this and this will be changed with #1708 again. If you prefer removing the string manipulation and not being compatible this is also totally fine for me.

Reply:
:+1: I have no strong opinion here :)